contents/docs/experiments/statistics-bayesian.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Bayesian statistics
+---
+
+PostHog's Bayesian analysis engine provides a statistically rigorous yet user-friendly approach to experimentation. Instead of p-values and confidence intervals, it calculates the **probability that one variant is better than another** and directly answers the question: "Should I ship this change?"
+
+For example, instead of saying "statistically significant at p < 0.05," you get intuitive statements like "There's a 96% chance that variant B increases conversion rate."
```

Comment:
```suggestion
For example, instead of saying "statistically significant at p < 0.05," you get intuitive statements like: 

> "There's a 96% chance that variant B increases conversion rate."
```

I just love the way this is written, I'd highlight it more